.github/workflows/rust.yml

```diff
@@ -17,8 +17,13 @@ jobs:
               run: |
                   set -e
 
+                  wget -qO - https://packages.lunarg.com/lunarg-signing-key-pub.asc | sudo apt-key add -
```

Comment:
don't use additional ppa for drivers, instead just use ubuntu docker container 
See https://github.com/membraneframework/video_compositor/commit/ce7f9572fb02c800bb790956c25f1103f7444e5b how to use containers